src/contracts/libraries/BeaconChainProofs.sol

```diff
@@ -75,6 +75,7 @@ library BeaconChainProofs {
     uint256 internal constant HISTORICAL_BATCH_STATE_ROOT_INDEX = 1;
 
     // in validator
+    uint256 internal constant VALIDATOR_PUBKEY_INDEX = 0;
     uint256 internal constant VALIDATOR_WITHDRAWAL_CREDENTIALS_INDEX = 1;
     uint256 internal constant VALIDATOR_BALANCE_INDEX = 2;
     uint256 internal constant VALIDATOR_SLASHED_INDEX = 3;
```

Comment:
should we add some of the structs to this library? e.g. the Validator container but in Solidity? even if we aren't directly using it (which we could, right?), it seems like it would help to be able to reference easily.
Just a thought I had, would be curious to hear others' thoughts.